create_db.sh

```diff
@@ -21,6 +23,14 @@ while true; do
     # The server is ready, so exit the loop
     break
   fi
+  # Increment the counter variable
+  counter=$((counter + 1))
+
+  if [[ $counter -ge $loop_limit ]]; then
```

Comment:
```suggestion
  if [[ ${counter} -ge ${loop_limit} ]]; then
```

Reply:
Hi @lgiordani , I have added this change in my next commit, is it ready to merge?